WalletWasabi.Fluent/Models/Transactions/PrivacySuggestionsModel.cs

```diff
@@ -90,16 +90,27 @@ public async Task<PrivacySuggestionsResult> BuildPrivacySuggestionsAsync(Transac
 
 	private IEnumerable<PrivacyItem> VerifyLabels(TransactionInfo info, BuildTransactionResult transactionResult)
 	{
-		var labels = transactionResult.SpentCoins.SelectMany(x => x.GetLabels(_wallet.AnonScoreTarget)).Except(info.Recipient);
-		var labelsArray = new LabelsArray(labels);
-
-		if (labelsArray.Any())
+		var pockets = _wallet.GetPockets();
+		var spentCoins = transactionResult.SpentCoins;
```

Comment:
`Private` and `Semi-Private` coins should be ignored as those coins cannot have labels, so the labels cannot be interlinked, but they are individual pockets so the condition can be `true` incorrectly.